egs2/qasr_tts/tts1/local/data.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+stage=-1
+stop_stage=2
+
+log "$0 $*"
+. utils/parse_options.sh
+
+
+#if [ $# -ne 0 ]; then
+#    log "Error: No positional arguments are required."
+#    exit 2
+#fi
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+
+db_root="downloads" 
```

Comment:
Why not use `db.sh` to set the directory? You can refer to other recipes to understand how to use it

Reply:
After you fix this issue and the CI issue, I will merge the PR~

Reply:
I fixed them